### PR TITLE
Fixes #5179 - Alt text for homepage img tags

### LINKED ISSUE #5179: Fix home page to pass accessibility tests

### PR DESCRIPTION
**Pull request by punamdahiya**

### CODE REVIEW

**Comment by punamdahiya** [comment]
Hold as we need to localize alt texts

**Comment by punamdahiya** [comment]
@6a68 PR fixes accessibility tests on homepage. Thanks!

File: locales/en-US/server.ftl [hunk 1]
```diff
@@ -9,6 +9,13 @@ gNoShots
 gScreenshotsDescription = Screenshots made simple. Take, save, and share screenshots without leaving Firefox.
 
 ## Header
+imgSettings =
+  .alt = Settings Icon
+imgSignIn =
+  .alt = Sign In Icon
```
Comment (flodolo) [inline locales/en-US/server.ftl]: "Sign In" is a verb, "sign-in" is the noun, and I believe you want the latter in this context

Comment (johngruen) [reply locales/en-US/server.ftl]: I think we should have empty alt strings here anyway since we have it on a the parent anchor

Comment (punamdahiya) [reply locales/en-US/server.ftl]: Created https://github.com/mozilla-services/screenshots/pull/5181 to use empty alt text

**Pull request merged**